src/main/java/com/kustudents/issuetracker/service/ImportanceService.java

```diff
@@ -0,0 +1,29 @@
+package com.kustudents.issuetracker.service;
+
+import java.util.List;
+
+import com.kustudents.issuetracker.model.entity.Importance;
+import com.kustudents.issuetracker.repository.ImportanceRepository;
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ImportanceService {
+
+    private final ImportanceRepository importanceRepo;
+
+    public List<Importance> getImportanceList(){
+        return importanceRepo.findAll();
+    }
+
+    public Importance getImportanceByID(Long ID){
+        return importanceRepo.getOne(ID);
+    }
+
+    public Page<Importance> getAllImportancesPaginatedAndFiltered(int page, int size, String orderBy, Boolean ascending){
+        return Page.empty();
```

Comment:
As I understand, this will be a TODO?

Reply:
Correct - need Fe first to see what fields will be needed before full implementation